databases/week1/dbweek1.sql

```diff
@@ -0,0 +1,111 @@
+-- Find out how many tasks are in the task table
+SELECT
+    COUNT(id)
+FROM 
+    task;
+
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT 
+    COUNT(id)
```

Comment:
COUNT(id) as task_amount
Pointer:
Provide a column name for columns you have created